packages/engine/src/avatar/functions/moveAvatar.ts

```diff
@@ -413,7 +418,7 @@ const _updateLocalAvatarRotationAttachedMode = (entity: Entity) => {
   const rigidbodyForward = new Vector3(0, 0, -1).applyQuaternion(rigidbody.targetKinematicRotation).setY(0)
   const angle = viewerForward.angleTo(rigidbodyForward)
 
-  if (angle > Math.PI * 0.25 || rotationNeedsUpdate == true) {
+  if (angle > Math.PI / 12 || rotationNeedsUpdate == true) {
```

Comment:
Why was this changed from multiplication to division?

Reply:
0.25 was much too high a value, it was sometimes not updating. we can change it back to division but it's only happening once or twice per frame